DEVELOPMENT.md

```diff
@@ -0,0 +1,66 @@
+| [@ceramicnetwork/cli](https://github.com/ceramicnetwork/js-ceramic/tree/develop/packages/cli) | [![npm](https://img.shields.io/npm/v/@ceramicnetwork/cli)](https://www.npmjs.com/package/@ceramicnetwork/cli) | Ceramic CLI and http daemon |
+| [@ceramicnetwork/common](https://github.com/ceramicnetwork/js-ceramic/tree/develop/packages/common) | [![npm](https://img.shields.io/npm/v/@ceramicnetwork/common)](https://www.npmjs.com/package/@ceramicnetwork/common) | Ceramic types and utilities |
+| [@ceramicnetwork/core](https://github.com/ceramicnetwork/js-ceramic/tree/develop/packages/core) | [![npm](https://img.shields.io/npm/v/@ceramicnetwork/core)](https://www.npmjs.com/package/@ceramicnetwork/core) | Ceramic protocol implementation |
+| [@ceramicnetwork/docid](https://github.com/ceramicnetwork/js-ceramic/tree/develop/packages/docid) | [![npm](https://img.shields.io/npm/v/@ceramicnetwork/docid)](https://www.npmjs.com/package/@ceramicnetwork/docid) | Encoding and decoding document identifiers |
+| [@ceramicnetwork/doctype-caip10-link](https://github.com/ceramicnetwork/js-ceramic/tree/develop/packages/doctype-caip10-link) | [![npm](https://img.shields.io/npm/v/@ceramicnetwork/doctype-caip10-link)](https://www.npmjs.com/package/@ceramicnetwork/doctype-caip10-link) | Interface for CAIP-10 link doctype |
+| [@ceramicnetwork/doctype-caip10-link-handler](https://github.com/ceramicnetwork/js-ceramic/tree/develop/packages/doctype-caip10-link-handler) | [![npm](https://img.shields.io/npm/v/@ceramicnetwork/doctype-caip10-link-handler)](https://www.npmjs.com/package/@ceramicnetwork/doctype-caip10-link-handler) | Handler for CAIP-10 link doctype |
+| [@ceramicnetwork/doctype-tile](https://github.com/ceramicnetwork/js-ceramic/tree/develop/packages/doctype-tile) | [![npm](https://img.shields.io/npm/v/@ceramicnetwork/doctype-tile)](https://www.npmjs.com/package/@ceramicnetwork/doctype-tile) | Interface for tile doctype |
+| [@ceramicnetwork/doctype-tile-handler](https://github.com/ceramicnetwork/js-ceramic/tree/develop/packages/doctype-tile-handler) | [![npm](https://img.shields.io/npm/v/@ceramicnetwork/doctype-tile-handler)](https://www.npmjs.com/package/@ceramicnetwork/doctype-tile-handler) | Handler for tile doctype |
+| [@ceramicnetwork/http-client](https://github.com/ceramicnetwork/js-ceramic/tree/develop/packages/http-client) | [![npm](https://img.shields.io/npm/v/@ceramicnetwork/http-client)](https://www.npmjs.com/package/@ceramicnetwork/http-client) | HTTP client that can interact with a remote Ceramic daemon |
+| [@ceramicnetwork/pinning-aggregation](https://github.com/ceramicnetwork/js-ceramic/tree/develop/packages/pinning-aggregation) | [![npm](https://img.shields.io/npm/v/@ceramicnetwork/pinning-aggregation)](https://www.npmjs.com/package/@ceramicnetwork/pinning-aggregation) |  |
```

Comment:
Can we also get descriptions for the pinning packages?  Sorry forgot to say this in last review round